ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -634,6 +634,26 @@ tests:
   steps:
     cluster_profile: ibmcloud
     workflow: openshift-e2e-ibmcloud-csi
+- as: e2e-aws-upi
```

Comment:
@jluhrsen has been working to include network slugs in names, e.g. #31128 and #31420.  Can you add `sdn` or `ovn` in here and your other jobs as appropriate?  Based on precedent like `as: e2e-metal-ipi-ovn`, I'd expect these names to look like `e2e-aws-upi-${NETWORK}`